rsf/sakai-rsf-web/templates/src/webapp/content/templates/sakai-date-field-input.html

```diff
@@ -33,7 +33,20 @@
 
 		// Initialize on DOM ready without jQuery
 		document.addEventListener('DOMContentLoaded', () => {
-			const initialVal = document.getElementById(rawHidden) ? document.getElementById(rawHidden).value : "";
+			let initialVal = document.getElementById(rawHidden) ? document.getElementById(rawHidden).value : "";
+			if (initialVal) {
+				// Check for ISO format with timezone. Could be more compact but ampersands don't work in RSF templates AFAIK.
+				var hasT = initialVal.indexOf('T') !== -1;
+				var hasPlus = (initialVal.lastIndexOf('+') == initialVal.length -3);
+				var hasMinus = (initialVal.lastIndexOf('-') == initialVal.length -3);
```

Comment:
can let be used or if they're not mutating use const?

Reply:
Done.